app.js

```diff
@@ -94,8 +98,12 @@ Test this function by hand in the console to get it working, and when you think
 
 // Write your code here
 function multiplyArray(testArray){ //eslint-disable-line
-
+  var testArray = [];
+  var testSum = multiply(multiply(2,3)[0],4);
```

Comment:
you need to use the values out of testArray to plug in this function, rather than the literal numbers. The idea is that no matter what those values are, you can use this function to calculate the product of this array.